playwright.config.ts

```diff
@@ -61,6 +61,9 @@ const config: PlaywrightTestConfig = {
       use: {
         ...devices['Desktop Chrome'],
         browserName: 'chromium',
+        launchOptions: {
+          args: ['--disable-gpu'],
+        },
```

Comment:
if we disable gpu the tests that involve  filtering will be super slow. is this necessary because of parallelization? or because the gpu on the instance is bad and trying to use it cause more issues than benefits?

Reply:
ah sorry
I forgot to remove this
I saw that qunit had this enabled and thought I needed it because some tests failed but this wasn't related
so forgot to remove it

Reply:
i ll remove it

Reply:
> i ll remove it

Tnx